nixos/doc/manual/man-nixos-rebuild.xml

```diff
@@ -521,7 +521,7 @@
 
    <varlistentry>
     <term>
-     <option>--flake</option> <replaceable>flake-uri</replaceable>[<replaceable>name</replaceable>]
+     <option>--flake</option> <replaceable>flake-uri</replaceable>#<replaceable>name</replaceable>
```

Comment:
```suggestion
     <option>--flake</option> <replaceable>flake-uri</replaceable>[#<replaceable>name</replaceable>]
```

The attribute is optional (it defaults to `nixosConfigurations.<hostname>`).

Reply:
I've wrapped it in `<optional>` which wraps with `[]` when rendered in manpages.